spydrnet_physical/util/ConnectPoint.py

```diff
@@ -171,13 +177,21 @@ def __str__(self) -> str:
         return "%5d %5d %5d %5d [%s]" % (self.from_x, self.from_y, self.to_x, self.to_y, self._level)
 
     def __mul__(self, scale):
+        '''Returns to and from coordinates after multipluing them with the scaling factor
+        '''
+        # But why not changing the original connectpoint
         pt = deepcopy(self)
         pt.from_connection = (scale*self.from_x, scale*self.from_y)
         pt.to_connection = (scale*self.to_x, scale*self.to_y)
         return pt
 
     def __rmul__(self, scale):
+        # reason for this method? Same code as __mul__ 
```

Comment:
`__mul__ `is called when you perform  `new_pt = pt * 2`
and `__rmul__ `is called when  `new_pt = 2 * pt `